### PR TITLE
[new release] sel (0.3.0)

### PR DESCRIPTION
**Pull request by gares**
Simple Event Library

- Project page: <a href="https://github.com/gares/sel">https://github.com/gares/sel</a>

##### CHANGES:

- Major code cleanup and documentation
- Removal of recurrent events
- APIs `pop` and `wait` can now be freely mixed
- CI on Linux, Windows and MacOS

### CODE REVIEW

File: packages/sel/sel.0.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "Simple Event Library"
+description: "Some sugar atop Unix.select"
```
Comment (mseri) [inline packages/sel/sel.0.3.0/opam:3]:
```suggestion
description: """This library is the result of our experience in using threads and the Lwt async monad to tame the problem of writing a server which has to listen and react to multiple sources of events. The library itself is sugar atop Unix.select. 
You can read more about the philosopy of the library on https://github.com/gares/sel
"""
This looks good to merge but I think the description could be a bit more informative
```

File: packages/sel/sel.0.3.0/opam [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+maintainer: ["Enrico Tassi"]
```
Comment (mseri) [inline packages/sel/sel.0.3.0/opam:4]: Would it be possible to have an email with the maintainer?

**Comment by gares** [comment]
done (also upstream)

**Comment by mseri** [comment]
I think if you don't use a multiline string, it needs to be all in one line. You can check it with `opam lint $filename.opam`

**Comment by gares** [comment]
Thanks for fixing, but it was dune generating that file.
Did I do anything wrong here: https://github.com/gares/sel/blob/main/dune-project ?

**Comment by mseri** [comment]
Actually I think dune generated a fine opam file, I don't understand why our CI is crashing during the lint right now

File: packages/sel/sel.0.3.0/opam [hunk 3]
```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
```
Comment (mseri) [inline packages/sel/sel.0.3.0/opam:1]:
```suggestion
```

**Comment by mseri** [comment]
Let's see if this works, there seems to be a repeated `opam-version` entry

**Comment by mseri** [comment]
Ok, that was the issue. The opam file in your repository looks good, there is no need to make further changes there

**Pull request merged**